clients/drcachesim/tracer/raw2trace.h

```diff
@@ -128,6 +128,8 @@ typedef enum {
     RAW2TRACE_STAT_EARLIEST_TRACE_TIMESTAMP,
     RAW2TRACE_STAT_LATEST_TRACE_TIMESTAMP,
     RAW2TRACE_STAT_FINAL_TRACE_INSTRUCTION_COUNT,
+    RAW2TRACE_STAT_KERNEL_INSTR_COUNT,
+    RAW2TRACE_STAT_SYSCALL_PT_TRACES_DECODED,
```

Comment:
Again thinking ahead: for interfaces where it's hard to rename things, I would not put "PT" in here.  Use a generic time like "KERNEL".

Reply:
For this stat, we do want to say syscall, if not syscall_pt. Renamed.